modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java

```diff
@@ -100,15 +106,24 @@ public String getCmd() {
 			_portletRequest, "customElementURLs", customElementURLs);
 
 		if (customElementURLs.length == 0) {
-			customElementURLs = new String[1];
+			customElementURLs = new String[] {
+				"https://liferay.github.io/liferay-frontend-projects" +
+					"/vanilla-counter/index.js"
```

Comment:
When going to: `https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js` it returns 404. I think it's not published yet

We could create a repo that publishes as a npm package and bring those files from unpkg

Reply:
We are waiting for IT to publish it (we don't have permissions). But once it is published we will just need to push changes to the `gh-pages` branch of `liferay-frontend-projects` (easier than using npm, unpkg, etc...).

Also, Brian wanted it to be under a Liferay domain, so we will publish it in GitHub, then point a liferay.com DNS entry to the site. Most probably.